src/table/components/TableHeadWrapper.jsx

```diff
@@ -34,15 +34,21 @@ function TableHeadWrapper({
   setFocusedCellCoord,
   keyboard,
 }) {
-  const headCellStyle = useMemo(() => getHeadStyle(layout, theme), [layout, theme.name()]);
-  const tableCellStyle = {
-    color: headCellStyle.color,
-    fontSize: headCellStyle.fontSize,
-    padding: headCellStyle.padding,
+  const tableBackgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
+  // When the table background color from sense theme is transparent, there is a default background color for the header
+  // to avoid seeing the table body through the header
+  const backgroundColor = tableBackgroundColor === 'transparent' ? '#FAFAFA' : tableBackgroundColor;
```

Comment:
This should be done inside getHeadStyle IMO. Then you can create `headStyle` after that

Reply:
actually would be nice if getHeadStyle returned both cell and head style

Reply:
Okay, I will work on these in #454

Reply:
Or just do it properly from the start? Since you are planning on merging this first?

Reply:
Sorry, I realized that merging this first is not a good idea. We should be planning on merging the rest.

Reply:
Done in #455